src/data/set/pointwise/basic.lean

```diff
@@ -6,6 +6,7 @@ Authors: Johan Commelin, Floris van Doorn
 import data.fin.tuple.basic
 import data.set.finite
 import algebra.module.basic
+import algebra.support
```

Comment:
Does this add a lot of imports? If so, this is maybe not great.

Reply:
Good remark. I've moved this to a new file.